package.json

```diff
@@ -89,6 +89,7 @@
     "interval-plus": "^1.0.2",
     "lodash.debounce": "^4.0.8",
     "mixpanel-browser": "^2.46.0",
+    "moment": "^2.29.4",
```

Comment:
We are not using `moment` lib as the date utils library, we use `date-fns`. 
Can you please use `date-fns`? Thanks!